crates/web5/src/dids/methods/did_dht/mod.rs

```diff
@@ -22,7 +22,7 @@ use crate::{
 };
 use std::sync::Arc;
 
-pub mod bep44;
+pub(crate) mod bep44;
```

Comment:
why not just omit `pub` entirely? doesn't look like we're using this anywhere else in the crate

Reply:
there are some public items inside the `bep44` crate which aren't in the APID. I guess we could add bep44 features to our APID, but I lean towards not doing so, for now at least

Reply:
I'd prefer not to add bep44 to the APID. I was thinking we could do `mod bep44` instead of `pub(crate) bep44` since I don't think we need bep44 outside of did_dht, unless I'm missing something

Reply:
oh right, I agree! done